ui/app/models/gcp/config.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { expandAttributeMeta } from 'vault/utils/field-to-attrs';
+
+export default class GcpConfig extends Model {
+  @attr('string') backend; // dynamic path of secret -- set on response from value passed to queryRecord
+
+  /* GCP config fields */
+  @attr({
+    label: 'Config TTL',
+    editType: 'ttl',
+    helperTextDisabled: 'The time-to-live (TTL) of generated tokens.',
```

Comment:
```suggestion
    helperTextDisabled: 'The TTL (time-to-live) of generated tokens.',
```

Reply:
Should this subtext [ttl](https://developer.hashicorp.com/vault/api-docs/secret/gcp#ttl)

> Specifies default config TTL for long-lived credentials (i.e. service account keys). Uses [duration format strings](https://developer.hashicorp.com/vault/docs/concepts/duration-format).